tmpl/blog/announcing-mirage-35-release.md

```diff
@@ -0,0 +1,101 @@
+# MirageOS 3.5.0 release
+
+We are happy to announce our MirageOS 3.5.0 release. We didn't announce post 3.0.0 releases too well -- that's why this post tries to summarize the changes in the MirageOS ecosystem over the past two years. MirageOS consists of over 100 opam packages, lots of which are reused in other OCaml projects and deployments without MirageOS. These opam packages are maintained and developed further by lots of developers.
+
+On the OCaml tooling side, since MirageOS 3.0.0 we did several major changes:
+- moved most packages to [dune](https://dune.build/) (formerly jbuilder) and began using [dune-release](https://github.com/samoht/dune-release) for smooth developer experience and simple releases,
+- require [opam](https://opam.ocaml.org) to be version 2.0.2 or later, allowing `pin-depends` in `config.ml`. `pin-depends` allows you to depend on a development branch of any opam package for your unikernel,
+- adjusted documentation to adhere to [odoc](https://github.com/ocaml/odoc/) requirements,
+- the `mirage` command-line utility now emits lower and upper bounds of opam packages, allowing uncompromising deprecation of packages,
+- support for OCaml 4.06.0 (and above), where `safe-string` is enabled by default. Strings are immutable now!!,
+- remove usage of `result` package, which has incorporated into `Pervasives` since OCaml 4.03.0.
+
+We are working on further changes (namely revising the `mirage` internal build system, at the moment using `ocamlbuild`, `ocamlfind`, `pkg-config`, and `make`) to make MirageOS more developer-friendly by using [dune](https://dune.build). Changing the internal build tooling allows for monorepos, incremental builds, cross-compilation, and other oft-requested improvements.
```

Comment:
This one's forward looking and so a bit odd in the release announcement for 3.5.0 -- perhaps remove it until the mirage 4 release? Or shift it towards the end in a 'next steps' section